CONTRIBUTING.md

```diff
@@ -1,28 +1,27 @@
-Contribution Agreement
-======================
+# Contribution Agreement
 
 As a contributor, you represent that the code you submit is your
 original work or that of your employer (in which case you represent
-you have the right to bind your employer).  By submitting code, you
+you have the right to bind your employer). By submitting code, you
 (and, if applicable, your employer) are licensing the submitted code
 to LinkedIn and the open source community subject to the BSD 2-Clause
 license.
 
-Responsible Disclosure of Security Vulnerabilities
-==================================================
+# Responsible Disclosure of Security Vulnerabilities
 
-Please do not file reports on Github for security issues.  Please
-review the guidelines on at (link to more info).  Reports should be
+Please do not file reports on Github for security issues. Please
+review the guidelines on at (link to more info). Reports should be
 encrypted using PGP (link to PGP key) and sent to
 security@linkedin.com preferably with the title "Github
-linkedin/<project> - <short summary>".
+linkedin/coral - <short summary>".
 
-Tips for Getting Your Pull Request Accepted
-===========================================
+# Tips for Getting Your Pull Request Accepted
 
 1. Make sure all new features are tested and the tests pass.
-2. Bug fixes must include a test case demonstrating the error that it
+2. Run `travis-build.sh` and make sure that it passes. Resolve any
```

Comment:
`travis-build.sh` does not exist in coral. Just use `./gradlew build` instead?

Reply:
Fixed.